src/components/HeroSection.jsx

```diff
@@ -0,0 +1,129 @@
+import PropTypes from "prop-types";
+
+const HeroSection = ({
+  className,
+  backgroundSrc,
+  overlayPrimaryFillColor,
+  overlaySecondaryFillColor,
+  overlayTertiaryFillColor,
+  conferenceStatus,
+  taglineText,
+  sloganTextColor,
+  sloganText,
+  title,
+  missionText,
+  eventDetailsBgColor,
+  eventDetailsText,
+  verse,
+  verseReferenceColor,
+  verseReference,
+}) => {
+  return (
+    <div className={className}>
+      <div className="relative">
+        <div className="overflow-hidden absolute inset-0 -z-10 rounded-bl-[5rem] md:rounded-bl-[10rem] xl:rounded-bl-[15rem]">
+          <img
+            alt="Background image"
+            className="object-cover absolute opacity-75 size-full grayscale"
+            src={backgroundSrc}
+          />
+          <BaseOverlay
+            primaryFillColor={overlayPrimaryFillColor}
```

Comment:
i tried playing around with changing the colors, and it also doesn't work at times like what i sent you in Whatsapp. not sure why that happens or is it just me? video is too big to upload here but here are screenshots:

<img width="986" alt="image" src="https://github.com/user-attachments/assets/5e479592-9877-46ae-a866-c4e1951102fc">
<img width="965" alt="image" src="https://github.com/user-attachments/assets/81877238-7c62-48e9-9420-34bcf5bbb9c6">

Reply:
this is because of how tailwind works, tailwind purges unused styles: https://tailwindcss.com/docs/content-configuration#class-detection-in-depth

Reply:
you can try changing it in the Storybook file if you want to test changing the colors